docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,132 @@
+
+### Data Docs
+
+Data Docs are human-readable documentation generated by GX.  Data Docs describe the standards that you expect your data to conform to, and the results of validating your data against those standards.  The Data Context manages the storage and retrieval of this information.
+
+You can configure where your Data Docs are hosted.  Unlike Stores, you can define configurations for multiple Data Docs sites.  You can also specify what information each Data Doc site provides, allowing you to format and provide different Data Docs for different use cases.
+
+- [Data Docs tutorial](/docs/guides/setup/configuring_data_docs/host_and_share_data_docs)
+
+## Datasources
+
+![Datasource overview](./images_overview/datasource_flowchart.png)
+
+Datasources connect GX to source data such as CSV files in a folder, a PostgreSQL database hosted on AWS, or any combination of data formats and environments. Regardless of the format of your source data or where it resides, Datasources provide GX with a unified API for working with it.
+
+- [Datasource tutorials](/docs/guides/connecting_to_your_data/connect_to_data_lp)
+
+### Data Assets and Batches
+
+Data Assets are collections of records within a Datasource.  While a Datasource tells GX how to connect to your source data, Data Assets tell GX how to organize that data. Although the records in your Data Assets can correspond directly to the contents of tables or files in your source data they do not necessarily need to. For instance, you could combine multiple tables worth of records in a SQL Datasource into a single Query Data Asset that joins the tables in question.  For a File Datasource, you could use regular expressions to define a Data Asset as the contents of all of the `.csv` files in a specific subfolder.
+
+Data Assets can be further partitioned into Batches.  Batches are unique subsets of records within a Data Asset.  For example, say you have a Data Asset in a SQL Datasource that consists of all records from last year in a given table.  You could then partition those records into Batches of data that correspond to the records for individual months of that year.
+
+- [Data Asset and Batch tutorials](/docs/guides/connecting_to_your_data/manage_data_assets_lp)
+
+### Batch Requests
+
+A Batch Request specifies one or more Batches within the Data Asset.  Batch Requests are the primary way of retrieving data for use in GX.  Because Batch Requests can retrieve multiple Batches they provide significant flexibility in how you utilize the data in a single Data Asset.
+
+As an example, GX can automate the process of running statistical analyzes for multiple Batches of data.  This is possible because you can provide a Batch Request that corresponds to multiple Batches in a Data Asset.  Alternatively, you can specify a single Batch from that same Data Asset so that you do not need to re-run the analyzes on all of your data when you are only interested in a single subset.
```

Comment:
```suggestion
As an example, GX can automate the process of running statistical analyses for multiple Batches of data.  This is possible because you can provide a Batch Request that corresponds to multiple Batches in a Data Asset.  Alternatively, you can specify a single Batch from that same Data Asset so that you do not need to re-run the analyses on all of your data when you are only interested in a single subset.
```